app/test_main.py

```diff
@@ -1,3 +1,27 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_get_human_age_return_zero_if_animals_age_is_zero() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_14_or_less_cat_or_dog_years_should_convert_into_0_human_age() -> None:
+    assert get_human_age(14, 14) == [0, 0]
+    assert get_human_age(12, 13) == [0, 0]
+
+
+def test_15_cat_or_dog_years_should_convert_into_1_human_age() -> None:
+    assert get_human_age(15, 15) == [1, 1]
+
+
+def test_27_cat_and_28_dog_years_should_convert_into_2_human_age() -> None:
+    assert get_human_age(27, 28) == [2, 2]
+
+
+def test_28_cat_and_29_dog_years_should_convert_into_2_human_age() -> None:
+    assert get_human_age(28, 29) == [3, 3]
```

Comment:
not appropriate names of the tests. 
Also tests should cover all the edge situations